### DISCUSSION TITLE
Is it possible to add Drupal 9 & 10 as quick add option

### DISCUSSION THREAD

**Discussion by mayankguptadotcom** [leokhoa/laragon#408] [Ideas]
Thank you for Laragon. I've seen it on various videos as well - but can't find an option that lets us setup Drupal 9 or Drupal 10 as a quick add option. Is it possible to add that?

**Comment by Pen-y-Fan** [comment]
Laragon is very flexible with customising the quick app menu.

I'm not familiar with Drupal, but I took a quick look at the docs and tried adding the following to the menu (Laragon menu > Quick app > Configuration...):

```text
# Drupal
Drupal=composer create-project drupal/recommended-project %s 
```

![Quick add](https://user-images.githubusercontent.com/40126936/218208649-7a7954b8-26ff-4601-9c45-7615a7cb95fd.png)

According to the [drupal download docs](https://www.drupal.org/download), after running the composer command above I need to run

```shell
cd drupal && php -d memory_limit=256M web/core/scripts/drupal quick-start demo_umami
```

🤷

I tried to use **quick app** without the command

- I used the project name  **drupal-test**, 
- The database **drupal-test** was automatically created.  (I would prefer **drupal_test** myself)
- The project was created in laragon\www\drupal-test

After installation and accepting the prompt to accept changes, by allowing elevated privileges. The website could be opened using the pretty URL <http://drupal-test.test>

![Drupal 10](https://user-images.githubusercontent.com/40126936/218209614-faa86dcc-c79a-47e2-9686-b5fbcd9a11c0.png)

For drupal 9 I suspect a specific version will need to be installed, try:

```text
Drupal-9=composer create-project 'drupal/recommended-project:^9.0' %s
```

Please let me know how you get on.

**Reply by rescenic** [reply]
Love it, thank you so much for this guide.

**Reply by aroon9002ahmed** [reply]
Thank you .. very easy
